package.json

```diff
@@ -12,10 +12,12 @@
     "@turf/turf": "6.3.0",
     "@visx/axis": "1.7.0",
     "@visx/curve": "1.7.0",
+    "@visx/event": "^1.7.0",
```

Comment:
It's a good idea to pin dependencies already when adding them. Can be achieved with setting `save-exact = true` in your NPM config.